tests/Doctrine/Tests/DBAL/Sharding/SQLAzure/AbstractTestCase.php

```diff
@@ -44,6 +44,7 @@ protected function setUp()
         // Customers, Orders, OrderItems federation tables.
         // See http://cloud.dzone.com/articles/using-sql-azure-federations
         $this->sm = new SQLAzureShardManager($this->conn);
+        var_dump($tmpConn->getSchemaManager()->listDatabases());
```

Comment:
Big fingers small phone (note maintainers can make changes to the pr, the check box for that is checked here)

I'll see if I can make the requested change.